src/components/accession2/NewTestModal.tsx

```diff
@@ -0,0 +1,38 @@
+import { Grid, Typography } from '@mui/material';
+import { Button, DialogBox } from '@terraware/web-components';
+import { Accession2 } from 'src/api/accessions2/accession';
+import strings from 'src/strings';
+
+export interface NewTestModalProps {
+  open: boolean;
+  accession: Accession2;
+  onClose: () => void;
+  reload: () => void;
+}
+
+export default function NewTestModal(props: NewTestModalProps): JSX.Element {
```

Comment:
`NewViabilityTestModal`? suggestion for file name and class name and where it's used.